.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,40 @@
+name: Lighthouse
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+    - 'content/**/*.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      # Build needs around 400 seconds on Netlify.
+      # We can not rely on "Waiting for 200", since the user
+      # might push another commit to an existing PR, meaning
+      # the deploy preview is already online.
+      - name: Sleep 400 seconds
+        run: sleep 400
```

Comment:
I re-ran it again 4 times after I have rebased.
The Netlify build times:
- 8 minutes
- 6min 15s
- 6min 27s
- 5min 

I think we have to go with something around 400 seconds = 6min 40s...

Reply:
Gotcha :(